beacon-chain/forkchoice/doubly-linked-tree/types.go

```diff
@@ -40,6 +40,7 @@ type Store struct {
 	highestReceivedSlot           types.Slot                            // The highest received slot in the chain.
 	receivedBlocksLastEpoch       [fieldparams.SlotsPerEpoch]types.Slot // Using `highestReceivedSlot`. The slot of blocks received in the last epoch.
 	allTipsAreInvalid             bool                                  // tracks if all tips are not viable for head
+	committeeBalance              uint64                                // tracks the average validator balance per slot: requires a lock on nodes to read/write
```

Comment:
I don't think this comment is right. Do you mean average committee balance? I wish my validator as that balance...

Reply:
ah yeah, it's the average total active validators per slot, will update in a sec